src/main/java/seedu/address/model/person/fields/InterviewTime.java

```diff
@@ -36,7 +37,20 @@ public InterviewTime(String interviewTime) {
      * @return True if the string is a valid interview time, false otherwise.
      */
     public static boolean isValidTime(String test) {
-        return test.equals("cancel") || test.matches(VALIDATION_REGEX);
+        if (test.equals("cancel")) {
+            return true;
+        } else {
```

Comment:
Might want to unnest the else part since the if body is already returning!

Reply:
Thank you for spotting!